### PR TITLE
Offers listing #2

### PR DESCRIPTION
**Pull request by jonzee**

### CODE REVIEW

File: src/sass/Global.scss [hunk 1]
```diff
@@ -55,13 +58,12 @@ ul {
...
   margin: auto;
   display: flex;
-  flex-wrap: wrap;
   box-sizing: border-box;
 }
 
 .raido {
   font-family: Junicode;
-  color: #AD000D;  
+  color: #AD000D;
```
Comment (michalmikolajczyk) [inline src/sass/Global.scss]: is this really the correct color?

File: src/sass/index.scss [hunk 2]
```diff
@@ -1,6 +1,7 @@
 @import 'AppBar';
-@import 'Cardgrid';
 @import 'Detail';
 @import 'Footer';
 @import 'Global';
```
Comment (michalmikolajczyk) [inline src/sass/index.scss]: the Global file should probably be handled as the first or the last one

File: src/ts/Offers/Card.tsx [hunk 3]
```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { Link } from 'react-router';
 import { FontIcon } from 'material-ui';
 
-export default function CardOwn(props) {
+export default function Card(props) {
```
Comment (michalmikolajczyk) [inline src/ts/Offers/Card.tsx]: perhaps `Offer` is a less generic name? Worried about possible conflicts with M-UI

File: src/ts/Offers/index.tsx [hunk 4]
```diff
@@ -1,7 +1,92 @@
 import * as React from 'react';
-import Cardgrid from './Cardgrid';
+import Container from '../Container';
+import Search from '../Search';
+import OffersList from './OffersList';
 
-export default class CardContainer extends React.Component<any, any> {
+const request = (url) =>
+  new Promise((res, rej) => {
```
Comment (michalmikolajczyk) [inline src/ts/Offers/index.tsx]: if this is returned, it should either be a one-liner, or use some type of bracket

File: src/ts/Offers/index.tsx [hunk 5]
```diff
@@ -17,31 +102,38 @@ export default class CardContainer extends React.Component<any, any> {
   }
 
   public loadMore() {
+    console.debug('load more results');
```
Comment (michalmikolajczyk) [inline src/ts/Offers/index.tsx]: remove `console` calls please. We can use `debug` instead from `npm`

File: src/ts/Offers/index.tsx [hunk 6]
```diff
@@ -17,31 +102,38 @@ export default class CardContainer extends React.Component<any, any> {
...
+          locationFilter={this.locationFilter}
+          selectFilter={this.selectFilter}
+          dateFilter={this.dateFilter}
+        />
+        <OffersList
+          data={this.state.data}
+          loading={this.state.loading}
+          loadMore={this.loadMore}
+        />
+      </Container>
```
Comment (michalmikolajczyk) [inline src/ts/Offers/index.tsx]: <3

File: src/ts/Search/Location.tsx [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+      load: false,
+    };
+    this.filter = props.filter;
+    this.initMap = this.initMap.bind(this);
+    this.displayData = this.displayData.bind(this);
+    this.onNewRequest = this.onNewRequest.bind(this);
+    this.onUpdateInput = this.onUpdateInput.bind(this);
+  }
+
+  public componentDidMount() {
```
Comment (michalmikolajczyk) [inline src/ts/Search/Location.tsx]: why `public`?

File: src/ts/Search/Select.tsx [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+    this.state = {select: []};
+    // creation of option from selectItems
+    this.createSelect(props.selectItems);
+    this.onChange = this.onChange.bind(this);
+  }
+
+  public createSelect(items) {
+    const createGroup = (values, group) =>
+      values.map((value, index) =>
+        (<div
```
Comment (michalmikolajczyk) [inline src/ts/Search/Select.tsx]: move the parentheses opening to the line above pls

File: src/ts/Search/Select.tsx [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+      :  <div className="selected empty">Dirtbike</div>
+  }
+
+  public render() {
+    return (
+      <div className="select">
+        <FontIcon className="material-icons">keyboard_arrow_down</FontIcon>
+        <div className="filter">
+          <SuperSelect
+            // select with multiple choice
```
Comment (michalmikolajczyk) [inline src/ts/Search/Select.tsx]: I don't think those comments work in JSX

File: src/sass/Global.scss (hunk 1 above)
Comment (jonzee) [inline src/sass/Global.scss]: there were some whitespace after

File: src/sass/index.scss (hunk 2 above)
Comment (jonzee) [inline src/sass/index.scss]: That's true, fixed.

File: src/ts/Offers/Card.tsx (hunk 3 above)
Comment (jonzee) [inline src/ts/Offers/Card.tsx]: Ok, renamed, indeed there is M-UI component `Card`.

File: src/ts/Offers/Offer.tsx [hunk 10]
```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { Link } from 'react-router';
 import { FontIcon } from 'material-ui';
 
-export default function Card(props) {
+export default function Offer(props) {
```
Comment (michalmikolajczyk) [inline src/ts/Offers/Offer.tsx]: 👍

File: src/ts/Offers/index.tsx [hunk 11]
```diff
@@ -2,8 +2,10 @@ import * as React from 'react';
 import Container from '../Container';
 import Search from '../Search';
 import OffersList from './OffersList';
+import debug from 'debug';
+var log = debug('app:Offers');
 
-const request = (url) =>
+const request = (url) => (
```
Comment (michalmikolajczyk) [inline src/ts/Offers/index.tsx]: 👍

File: src/ts/Search/Select.tsx [hunk 12]
```diff
@@ -16,9 +16,9 @@ export default class Select extends React.Component<any, any> {
   }
 
   public createSelect(items) {
-    const createGroup = (values, group) =>
-      values.map((value, index) =>
-        (<div
+    const createGroup = (values, group) => (
+      values.map((value, index) => (
```
Comment (michalmikolajczyk) [inline src/ts/Search/Select.tsx]: 👍

**Pull request merged**